app/src/main/java/com/pileproject/drive/programming/visual/block/BlockFactory.java

```diff
@@ -38,131 +38,93 @@
     public static final int SEQUENCE = 0;
     public static final int REPETITION = 1;
     public static final int SELECTION = 2;
+    @Deprecated
     public static final int UNDO = 3;
     public static final int LOAD = 4;
 
-    /**
-     * This class can't be created as an instance
-     */
     private BlockFactory() {
     }
 
-    /**
-     * Return class by name
-     *
-     * @param className
-     * @return Class
-     */
     @SuppressWarnings("unchecked")
-    private static <T> Class<T> getClassForName(String className) throws RuntimeException {
+    private static <T extends BlockBase> Class<T> getClassForName(String className) throws RuntimeException {
+
         try {
             return (Class<T>) Class.forName(className);
+
```

Comment:
I'm sorry, I didn't want to mean it :( I agree that adequate blank lines help code reading.

I couldn't see the consistency between these blank lines here and previous codes at review but now I'm guessing you added blank lines between long lines. It can give a kind of tender impression to codes I think. I wanted to know some of rules, i.e. why you added it here and didn't there.

We haven't talked enough about blank lines beforehand and formatters never specify it. I really appreciate if you share your thoughts :)